xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStreamPVRManager.cpp

```diff
@@ -383,9 +387,7 @@ bool CDVDInputStreamPVRManager::CloseAndOpen(const char* strFile)
   return false;
 }
 
-bool CDVDInputStreamPVRManager::SupportsChannelSwitch(void)
+bool CDVDInputStreamPVRManager::IsOtherStreamHack(void)
 {
-  PVR_CLIENT client;
-  return g_PVRClients->GetPlayingClient(client) &&
-         client->HandlesInputStream();
+  return m_isOtherStreamHack;
```

Comment:
which case am I missing?

* no other stream: m_pOtherStream = NULL, m_isOtherStreamHack = false
* input stream failed to create: m_pOtherStream = NULL, m_isOtherStreamHack = false
* input stream failed to open: m_pOtherStream is deleted and NULLed, m_isOtherStreamHack = false
* other stream opened: m_pOtherStream != NULL, m_isOtherStreamHack = true